README.md

```diff
@@ -25,3 +25,13 @@ Link Visual Studio Code to that database by going to ```View``` and selecting th
 
 If all went correctly, your database should be connected, and you can then run the project.
 To run the project, press the green "play" triangle that is above the code editor, roughly in the middle.
+
+# Human Resources
+
+## Departments
+
+###View All
+The great folks in the Human Resources Department can view all Departments when the Department Naviagation Link is clicked.

```

Comment:
There is only a details option... will there be more ? What are the options and what do they do?